torch_geometric/data/summary.py

```diff
@@ -70,12 +82,26 @@ def from_dataset(
             num_nodes_list.append(data.num_nodes)
             num_edges_list.append(data.num_edges)
 
-        return cls(
-            name=name,
-            num_graphs=len(dataset),
-            num_nodes=Stats.from_data(num_nodes_list),
-            num_edges=Stats.from_data(num_edges_list),
-        )
+        nodes_per_type = []
+        edges_per_type = []
+        if per_type_breakdown:
+            for node_t in node_types:
+                node_list = []
+                for data in dataset:
+                    node_list.append(data.get_node_store(node_t).num_nodes)
+                nodes_per_type.append([node_t, Stats.from_data(node_list)])
```

Comment:
Just for consistency 😄 if it's `edges_list`, let's also make it `nodes_list`
```suggestion
                nodes_list = []
                for data in dataset:
                    nodes_list.append(data.get_node_store(node_t).num_nodes)
                nodes_per_type.append([node_t, Stats.from_data(nodes_list)])
```